DOCS/man/input.rst

```diff
@@ -3438,6 +3444,35 @@ Property list
 ``current-ao``
     Current audio output driver (name as used with ``--ao``).
 
+``clipboard`` (RW)
+    Access to the system clipboard as a key/value map of types and data.
+
+    Sub-paths can be accessed directly; e.g. ``clipboard/text`` can be read, written,
+    or observed.
+
+    The top-level object itself can be written directly as a shortcut; the intended
+    type will be guessed.
```

Comment:
I must have missed where this is implemented but also: is that a good idea?

Reply:
Maybe we should just say "Setting `clipboard` to a string will write to the text clipboard as a shortcut"? And it's this code: https://github.com/mpv-player/mpv/pull/13837/commits/3a851741e9d72ac9080b06d20ddda1308331b8e1#diff-830516050a5bf883d9c265fa0cf3c5b59d23d8bc9e6dec63db3d3d9b9868bb6bR4016-R4025

Reply:
Ah, I see. I expected to find something like "if `://` present then write as URL".

> Maybe we should just say "Setting clipboard to a string will write to the text clipboard as a shortcut"

sure